conf/modules.config

```diff
@@ -20,164 +19,117 @@ process {
+            genome_reference == 'grch37'                                ? "--genome_reference 'https://grch37.ensembl.org/'" : '',
+            genome_reference == 'grch38'                                ? "--genome_reference 'https://www.ensembl.org'" : '',
+            params.proteome_reference ? "--proteome_reference ${params.proteome_reference}" : "",
+            params.fasta_output ? "--fasta_output" : "",
         ].join(' ').trim()
         publishDir = [
-            path: { "${params.outdir}/split_predictions/${meta.sample}" },
-            mode: params.publish_dir_mode
+            path: { "${params.outdir}/epytope" },
+            mode: params.publish_dir_mode,
+            pattern: '*.tsv'
         ]
     }
-
-    withName: EPYTOPE_PEPTIDE_PREDICTION_PEP {
-        // Argument list needs to end with --peptides
-        ext.args   = [
-                    genome_reference != 'grch37' & genome_reference != 'grch38' ? "--genome_reference '${genome_reference}'" : '',
-                    genome_reference == 'grch37'                                ? "--genome_reference 'https://grch37.ensembl.org/'" : '',
-                    genome_reference == 'grch38'                                ? "--genome_reference 'https://www.ensembl.org'" : '',
-                    '--peptides'
-        ].join(' ').trim()
+    withName: CAT_FASTA {
+        ext.prefix = {"${meta.sample}"}
         publishDir = [
-            path: { "${params.outdir}/split_predictions/${meta.sample}" },
-            mode: params.publish_dir_mode
+            path: { "${params.outdir}/epytope" },
+            mode: params.publish_dir_mode,
+            pattern: '*.fasta'
         ]
     }
 
-    withName: EPYTOPE_PEPTIDE_PREDICTION_VAR {
-        // Argument list needs to end with --somatic_mutation
-        ext.args   = [
-                    genome_reference != 'grch37' & genome_reference != 'grch38' ? "--genome_reference '${genome_reference}'" : '',
-                    genome_reference == 'grch37'                                ? "--genome_reference 'https://grch37.ensembl.org/'" : '',
-                    genome_reference == 'grch38'                                ? "--genome_reference 'https://www.ensembl.org'" : '',
-                    '--somatic_mutation'
-        ].join(' ').trim()
-        publishDir = [
-            path: { "${params.outdir}/split_predictions/${meta.sample}" },
-            mode: params.publish_dir_mode
-        ]
+    withName: FASTA2PEPTIDES {
+        publishDir = [ enabled: false ]
     }
 
-    withName: MERGE_JSON_SINGLE {
-        ext.args = " --single_input "
+    withName: SPLIT_PEPTIDES {
+        publishDir = [ enabled: false ]
     }
 
-    withName: MERGE_JSON_MULTI {
-        ext.args = " --input \$PWD "
+    withName: PREPARE_PREDICTION_INPUT {
+        ext.prefix = {"${meta.file_id}"}
+        publishDir = [ enabled: false ]
     }
 
-    withName: CAT_TSV {
+    withName: MHCFLURRY {
```

Comment:
Combine the processes MHCFLURRY and MHCNUGGETS

Reply:
Why not keeping them separate bc of different output folder?